src/pages/Calendar/Calendar-Day.vue

```diff
@@ -0,0 +1,393 @@
+<script>
+import FlowRunMenu from '@/pages/Calendar/RunMenu'
+import { mapGetters } from 'vuex'
+import { formatTime } from '@/mixins/formatTimeMixin'
+import ExternalLink from '@/components/ExternalLink'
+
+export default {
+  components: {
+    FlowRunMenu,
+    ExternalLink
+  },
+  filters: {},
+  mixins: [formatTime],
+  props: {
+    calendarInterval: {
+      required: true,
+      type: Number
+    },
+    date: {
+      required: true,
+      type: String
+    },
+    flowId: {
+      required: true,
+      type: String
+    },
+    type: {
+      required: true,
+      type: String
+    }
+  },
+  data() {
+    return {
+      skip: false,
+      loadingKey: 0,
+      gettingRuns: false,
+      flowRunEvents: [],
+      selectedEvent: null,
+      selectedOpen: false,
+      selectedElement: null,
+      upcoming: [],
+      intervalHeight: 100,
+      scheduleBanner: false,
+      closeBanner: false
+    }
+  },
+  computed: {
+    ...mapGetters('api', ['isCloud']),
+    ...mapGetters('tenant', ['tenant']),
+    ...mapGetters('user', ['timezone']),
+    intervalCount() {
+      return (60 / this.calendarInterval) * 24
+    },
+    start() {
+      let days = 1
+      const start =
+        this.tzOffset(this.date) < 0
+          ? this.subtractDay(this.date, days)
+          : this.date
+      return start
+    },
+    end() {
+      let days = 1
+      switch (this.type) {
+        case '4day':
+          days = this.tzOffset(this.date) > 0 ? 5 : 4
+          break
+        case 'day':
+          days = this.tzOffset(this.date) > 0 ? 2 : 1
+          break
+      }
+      return this.addDay(this.date, days)
+    }
+  },
+  watch: {
+    async date() {
+      this.selectedEvent = null
+      this.flowRunEvents = await this.flowRunEventsList()
+      this.scrollToElement()
+    },
+    async flowId() {
+      this.selectedEvent = null
+      this.flowRunEvents = await this.flowRunEventsList()
+      this.scrollToElement()
+    }
+  },
+  async created() {
+    this.flowRunEvents = await this.flowRunEventsList()
+    this.scrollToElement()
+  },
+  methods: {
+    async flowRunEventsList() {
+      this.gettingRuns = true
+      const scheduledRuns = await this.$apollo.query({
+        query: require('@/graphql/Calendar/calendar-day-scheduled-flow-runs.gql'),
+        variables: {
+          startTime: this.start,
+          endTime: this.end,
+          flowId: this.flowId
+        },
```

Comment:
It might be cleaner to define these once at the top of the method and then reference them in each of the queries, something like this:
```vue
    const queryVariables = {
      startTime: this.start,
      endTime: this.end,
      flowId: this.flowId
    }

    const finishedRuns = await this.$apollo.query({
        query: require('@/graphql/Calendar/calendar-day-flow-runs.gql'),
        variables: queryVariables,
        loadingKey: 'loadingKey'
    })
```

We could also collect the queries using `Promises.all` to let these run in parallel instead of sequentially.